node/service/src/chain_spec/frequency.rs

```diff
@@ -111,9 +111,9 @@ pub mod foundation_keys {
 		"0xf811cbc8611b38f68f4e486284e71eb45523fc5cf685c070bc76331d595cf81c";
 }
 
-// pub fn load_frequency_spec() -> Result<ChainSpec, String> {
-// 	ChainSpec::from_json_bytes(&include_bytes!("../../specs/frequency.json")[..])
-// }
+pub fn load_frequency_spec() -> ChainSpec {
+	ChainSpec::from_json_bytes(&include_bytes!("../../../../resources/frequency.json")[..]).unwrap()
+}
 
 pub fn frequency() -> ChainSpec {
```

Comment:
Want to delete this function?

Reply:
@wilwade decided not yet, we will take it out in an update even though it's not being used.

Reply:
Yes. We will take it out in a future PR. I wanted to make sure there was a commit that we could reference where both existed.